app/views/ShareView/index.tsx

```diff
@@ -327,6 +355,82 @@ class ShareView extends Component<IShareViewProps, IShareViewState> {
 			this.messageComposerRef.current?.setInput(item.description || '');
 		}
 	};
+	/**
```

Comment:
Since none of the compression methods deal with state management, you should remove them from `ShareView/index.tsx` and move them to `ShareView/utils.ts`.